decidim-elections/app/cells/decidim/votings/content_blocks/landing_page/header/show.erb

```diff
@@ -0,0 +1,29 @@
+<section class="extended hero" style="background-image:url(<%= current_participatory_space.banner_image.url %>);">
+  <div class="hero__container">
+    <div class="row">
+      <div class="columns large-10 text-left">
+        <h1 class="text-highlight hero-heading mb-none">
+          <%= translated_attribute(current_participatory_space.title) %>
+        </h1>
+
+        <h3 class="text-highlight">
+          <%= voting_dates %>
+        </h3>
+      </div>
+    </div>
+
+    <% if !translated_button_text.blank? && !translated_button_url.blank? %>
+      <div class="row">
+        <div class="row column medium-8 mt-m">
+          <div class="column medium-3">
+            <%= cta_button %>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
+</section>
+
+<section class="section">
+  <%= extended_navigation_bar(navigation_items) %>
```

Comment:
The navigation bar is not in the PR screenshot, but it will be there, right?

Reply:
Yes, it has the participatory space nav bar.
I've updated the screenshot.